packaging/build_px4fw.sh

```diff
@@ -18,14 +18,14 @@ else
 
         # for our own HW, use HW specific siging. For pixhawks, and icicle board,
         # use the PX4 default signing script and keys
-        if [[ $NAME = saluki* ]]
-        then
-            export SIGNING_TOOL=boards/ssrc/saluki-v1/tools/ed25519_sign.py
-            export SIGNING_ARGS=boards/ssrc/$NAME/tools/ed25519_test_key.pem
-        else
+        #if [[ $NAME = saluki* ]]
```

Comment:
Removing this will cause saluki-pi stop booting, as it uses different keys than the saluki-v1/v2